contracts/BatchExchangeViewer.sol

```diff
@@ -322,12 +336,13 @@ contract BatchExchangeViewer {
             uint128 priceDenominator,
             uint128 usedAmount
         ) = abi.decode(order, (uint16, uint16, uint32, uint32, uint128, uint128, uint128));
+        // Unconditionally serialize user address to not break pagination
+        copyInPlace(abi.encodePacked(user), target, offset);
```

Comment:
This was a problem before as well right?

Reply:
Yes, it was introduced in #663 (so not yet deployed) and discovered by my change in unit tests (maybe I should add a regression test just for this one). 

I thought I could cover it up here, but you caught me. 👮

Reply:
I'm fine with including it here as it is a related change and not yet deployed.

Reply:
Given we need to do an emergency deployment (#672), I'm backing out the bad commit and will resubmit it with the fix where it belongs to.